src/main/java/heronarts/lx/command/LXCommand.java

```diff
@@ -1759,177 +1759,7 @@ public boolean isIgnored() {
-
-      @Override
-      public void undo(LX lx) {
-        lx.engine.snapshots.moveSnapshot(this.snapshot.get(), this.fromIndex);
-      }
-    }
-
-    public static class RemoveSnapshot extends RemoveComponent {
-
-      private final ComponentReference<LXSnapshot> snapshot;
-      private final JsonObject snapshotObj;
-      private final int snapshotIndex;
-
-      public RemoveSnapshot(LXSnapshot snapshot) {
-        super(snapshot);
-        this.snapshot = new ComponentReference<LXSnapshot>(snapshot);
-        this.snapshotObj = LXSerializable.Utils.toObject(snapshot);
-        this.snapshotIndex = snapshot.getIndex();
-      }
-
-      @Override
-      public String getDescription() {
-        return "Delete Snapshot";
-      }
-
-      @Override
-      public void perform(LX lx) {
-        lx.engine.snapshots.removeSnapshot(this.snapshot.get());
-      }
-
-      @Override
-      public void undo(LX lx) throws InvalidCommandException {
-        LXSnapshot snapshot = new LXSnapshot(lx);
-        snapshot.load(lx, this.snapshotObj);
-        lx.engine.snapshots.addSnapshot(snapshot, this.snapshotIndex);
-        super.undo(lx);
-      }
-    }
-
-    public static class Update extends LXCommand {
-
-      private final ComponentReference<LXSnapshot> snapshot;
-      private JsonObject previousState;
-
-      public Update(LXSnapshot snapshot) {
-        this.snapshot = new ComponentReference<LXSnapshot>(snapshot);
-      }
-
-      @Override
-      public String getDescription() {
-        return "Update Snapshot";
-      }
-
-      @Override
-      public void perform(LX lx) throws InvalidCommandException {
-        this.previousState = LXSerializable.Utils.toObject(lx, this.snapshot.get());
-        this.snapshot.get().update();
-      }
-
-      @Override
-      public void undo(LX lx) throws InvalidCommandException {
-        this.snapshot.get().load(lx, this.previousState);
-      }
-
-    }
-
-    public static class Recall extends LXCommand {
-
-      private final ComponentReference<LXSnapshot> snapshot;
-      private final List<LXCommand> commands = new ArrayList<LXCommand>();
-      private boolean recalled = false;
-
-      public Recall(LXSnapshot snapshot) {
-        this.snapshot = new ComponentReference<LXSnapshot>(snapshot);
-      }
-
-      @Override
-      public void perform(LX lx) {
-        this.commands.clear();
-        this.recalled = lx.engine.snapshots.recall(this.snapshot.get(), this.commands);
-      }
-
-      @Override
-      public void undo(LX lx) throws InvalidCommandException {
-        for (LXCommand command : this.commands) {
-          command.undo(lx);
-        }
-      }
-
-      @Override
-      public boolean isIgnored() {
-        return !this.recalled;
-      }
-
-      @Override
-      public String getDescription() {
-        return "Recall Snapshot";
-      }
-    }
-
-    private static class RemoveView extends LXCommand {
-
+     private static class RemoveView extends LXCommand {
```

Comment:
These classes were specifically all inside `LXCommand` so that when referenced elsewhere there is a consistent class naming scheme that makes it clear what type of command it is.

e.g. `LXCommand.Snapshots.RemoveView` rather than `RemoveView`

This change arbitrarily moves just a few of many different commands out of this naming hierarchy. I think it becomes particularly less readable for a class like `Update` - which is quite semantically clear as `LXCommand.Snapshot.Update` (this updates a snapshot), vs. someone reading `new Update` is going to really wonder what this means.

These were all grouped as such for that reason.